src/components/dropdowns/ProfileDropdown.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="relative ml-3" @click="toggleMenu">
+    <div>
+      <button
+        class="relative flex rounded-full bg-gray-800 text-sm focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-gray-800"
+      >
+        <span class="absolute -inset-1.5" />
+        <span class="sr-only">Open user menu</span>
+        <img class="h-8 w-8 rounded-full" :src="userImage" alt="" />
```

Comment:
add fallback div to display profile initials or empty circle in case of missing userImage